ui/pages/confirmations/utils/confirm.ts

```diff
@@ -23,3 +25,9 @@ const SIGNATURE_APPROVAL_TYPES = [
 export const isSignatureApprovalRequest = (
   request: ApprovalRequest<Record<string, Json>>,
 ) => SIGNATURE_APPROVAL_TYPES.includes(request.type as ApprovalType);
+
+export const parseTypedDataMessage = (dataToParse: string) => {
```

Comment:
Minor, but should we have unit tests for these utils or will they be entirely tested by the component tests?

Reply:
Good point, let me add those.